plugin/minimal/README.md

```diff
@@ -0,0 +1,74 @@
+# minimal
+
+## Name
+
+*minimal* - minimizes the size of the DNS query response 
+
+## Description
+
+The minimal plugin controls the addition of records to the AUTHORITY and ADDITIONAL section of the dns query response.
+
+## Syntax
+
+~~~
+minimal yes | no
+~~~
+
+## Examples
+
+~~~ corefile
+. {
+    minimal yes
+    forward . 8.8.8.8 
+}
+~~~
+
+## Design and Implementation details
```

Comment:
entire section can be removed - this does not need to be in the readme. Add it to the PR or as a comment on the issue so we can review/discuss.

Reply:
Yes. I intended to add that for review and discussion. I planned to remove that for the final PR. I shall seperate out the design and implementation sections.